### ISSUE TITLE
mod still being worked on?

### ISSUE THREAD

**Issue by AerowynX** [KozGit/DOOM3-BFG-RIFT#1]
wasn't sure were else to ask was curious if this was still being worked on

**Comment by KozGit** [comment]
Yep -  Haven't had a lot of time, have had ( a lot ) of family in town for the holidays.  Today is the last day they are here, so should be able to finish testing.  I've updated some things:

Sorted out the focus issue that was keeping people from re-binding controls correctly using the controllers.

Included the correct default controller layout config instead of the right hand only config packaged in the 
prior release. ( I really duffed that one, that config was from an experimental build where the right controller is attached to a TopShot elite and the left is tucked in my belt for directional chest tracking, was never supposed to see the light of day. )

Added a bindable control to bring  up the pause/system menu via controller button

Fixed the movePoint control method so it works with all movement directions, not just forward ( works like onward now), with options on how to orient the player body (if viewing it), either to the point direction or the look direction

Added a click to move option. 

Fixed the glitchy intro menus - vsync issue that interestingly enough did not occur on my win 7 dev box.  Which oddly enough fried the day before Thanksgiving.  One old GTX780 bites the dust.

Fixed the scale in the intro menus.

Added motion control crouching.

Fixed latency issue caused when SMP enabled

Smoothed out going over stairs

There were a bunch of other small fixes.  I'm adding some options for how pad input is processed as well.

Hopefully will have something to release in the next few days or so.  If you have any requests or suggestions I'd love to hear them, thx!

**Comment by AerowynX** [comment]
Awesome so with click to move will you be able to set touch to walk then click to run?

**Comment by Aesop112** [comment]
Melee and two handed gun handleing ?

**Comment by KozGit** [comment]
Awesome so with click to move will you be able to set touch to walk then click to run?

I'm sorry, I'm not sure exactly what you mean by this.. you can set it so just touching the touchpad will make you walk, and clicking the pad ( or whatever button you define ) will make you run, is that what you mean?  I also added where you wont walk at all unless you click while touching the pad, and if you click again while moving you toggle run.  Adding these type of options are pretty straightforward, so let me know if there is something else I'm forgetting and I'll see what I can do.

**Comment by AerowynX** [comment]
So yea I'd like just touching the trackpad to have me walk and pushing in on it to run...this is how I play with other mod and works well

**Comment by KozGit** [comment]
Ok, that's no problem, you just leave the axes on the left pad bound to the player move directions, and the pad press bound to run - this is actually the new default scheme for the left controller.

**Comment by puttis86** [comment]
Is the flashlight time controlled by your code somewhere? I find the very short uptime and recharge time of the flashlight super annoying and immersion breaking. (I'm on mars shooting plasma weapons but my stupid light cant be on for more then a few seconds? wtf...) and went digging in the gamefiles to try and change it. I could alter the behavior of the guns with ease, but the flashlight stays the same what ever values I give it. Every bit of help pointing me in the right direction would be much appreciated!

**Comment by AerowynX** [comment]
the console command flashlight_batteryDrainTimeMS   can adjust that.. try -1 for infinite

**Comment by puttis86** [comment]
you sexy devil you!

**Comment by AerowynX** [comment]
i heard some people having issues with - 1 if you do just set a high number in there

**Comment by puttis86** [comment]
yeah -1 didnt work, but 0 or a high enough number worked wonders! than you dear sir!

**Comment by AerowynX** [comment]
i think you need a space between the - and the 1 for it to work either way glad it's working for you

**Comment by puttis86** [comment]
ah, ill keep that in mind if it goes tits up in the future!

**Comment by AerowynX** [comment]
you know of anyway to get aliasing to look better in this game? MSAA and SS don't really seem to have much effect.. for example at the start look at the railing going up the stairs as you walk the shimmering is awful and I found even at 8x MSAA or 1.5x resolution scaling it still is pretty bad.. all that AA seems to help in this game is text readability

**Comment by tkapis** [comment]
I am eagerly waiting for the update to play the game as at the moment no menu button is a big problem. Any idea when it will be released?

**Comment by KozGit** [comment]
I received my touch today, and took some time tonight to implement native rift/touch support via the Oculus SDK.  I want to do a little testing in the morning, and should post test binaries for both SteamVR and native Oculus tomorrow in the afternoon or evening. Sorry for the delay!

**Comment by tkapis** [comment]
Great, I think it's fantastic that you made this mod which is very immersive with the pda and the touch interface. Looking forward to playing with the update and thanks for this.

**Comment by KozGit** [comment]
I posted an update with native Oculus and Touch support, you can download it on the release page.  The OpenVR version has had all the changes implemented, I just need to package it up and post it, i'll do that tomorrow.

Aerowyn, I'm not sure why changing the pixel density or MSAA settings is not having an effect for you, normally the change is pretty drastic from pixel density 1.0 to 1.5.  If you go too high though, it can actually start to have a negative impact on image quality.  I like to run with 2x msaa enabled and at a pixel density around 1.4 or 1.5.  (Are you having the same issue on both mods?)

**Comment by KozGit** [comment]
Allright, I've posted an updated version of the OpenVR version of the mod that should hopefully address the issues some people were having with controls, and implemented some new features. Check the release page for info and let me know how it works for you!

**Comment by AerowynX** [comment]
Awesome thanks man!

**Comment by KozGit** [comment]
Hi,

Sorry for taking so long to address these issues.

There is a new repository for the mod  - you can download an updated version for OpenVR here:

https://github.com/KozGit/DOOM-3-BFG-VR/releases

I apologize, but the OpenVR version has been upgraded, but not tested, so expect some issues.  PLEASE try running the mod in a clean install first, before installing the texture pack.  This will make it much easier to sort out any issues. 
 
If you have any issues, ( and i expect some ), please open an issue in the new repository and I'll get to work on fixing them, I'm not adding any more features until the bugs are worked out of this.

Thx!
